src/modules/ie-objects/ObjectDetailPage.tsx

```diff
@@ -1503,15 +1503,26 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 			locale,
 			publicRuntimeConfig.CLIENT_URL
 		).filter(({ data }: MetadataItem): boolean => !!data);
-		const rightsAttributionText = compact([
-			getIeObjectCreatorAsText(mediaInfo),
-			mediaInfo.datePublished,
-			mediaInfo.name,
-			mediaInfo.maintainerName,
-			getIeObjectRightsStatusAsString(mediaInfo),
-			publicRuntimeConfig.CLIENT_URL +
-				ROUTES_BY_LOCALE[locale].permalink.replace(':pid', mediaInfo.schemaIdentifier),
-		]).join(', ');
+		let rightsStatusInfo = null;
```

Comment:
Is it necessary to use a let here? Personally I like to use const if is not reassigned more than 1 times in the code. I would do something like:
` const rightsStatusInfo = mediaInfo?.dctermsFormat === IeObjectType.Newspaper ? getIeObjectRightsStatusInfo(mediaInfo, locale) : null`

Reply:
als de tertiare operator meer dan 1 regel neemt, dan split ik die meestal op met een let + if voor de duidelijkheid.
Maar inderdaad, dan blijft de variable wel mutable voor de rest van de code.

Mss is dit de betere code dan?
```
const isNewspaper = mediaInfo?.dctermsFormat === IeObjectType.Newspaper
const rightsStatusInfo = isNewspaper ? getIeObjectRightsStatusInfo(mediaInfo, locale) : null;
```